static/js/queues.js

```diff
@@ -125,7 +125,8 @@ document.querySelector('#declare').addEventListener('submit', function (evt) {
     arguments: Dom.parseJSON(data.get('arguments'))
   }
   HTTP.request('PUT', url, { body })
-    .then(() => {
+    .then((response) => {
+      if (response?.is_error) { return }
```

Comment:
Could we just check for `response.reason` here instead? It should be set on error, and we wouldn't need to set `is_error` as well.

Reply:
not really, we don't return a reason so the response is undefined (if it is a bad request) or null (if it is ok). (see http.js)

Reply:
Okay. It looks to me like we always set a reason (either to `response.statusText` or `json.reason`), but maybe statusText doesn't always exist.

Reply:
Oh yes, sorry in `http.js` we set the reason but the json conversion dosn't work for this prticular endpoint because we don't return anything from the endpoint `put api/queues`